backend/Makefile

```
@@ -1,8 +1,11 @@
-.PHONY: dev build run tag seeds test format
+.PHONY: dev claude build run tag seeds test format
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)